kbfsmd/root_metadata_v3_test.go

```diff
@@ -52,6 +51,52 @@ func TestRootMetadataVersionV3(t *testing.T) {
 	check(tlf.Public, tlf.TeamKeying, ImplicitTeamsVer)
 }
 
+func TestRootMetadataHardcodedV3(t *testing.T) {
+	tlfID := tlf.FakeID(1, tlf.Private)
+
+	uid := keybase1.MakeTestUID(1)
+	bh, err := tlf.MakeHandle(
+		[]keybase1.UserOrTeamID{uid.AsUserOrTeam()}, nil, nil, nil, nil)
+	require.NoError(t, err)
+
+	brmd, err := MakeInitialRootMetadataV3(tlfID, bh)
+	require.NoError(t, err)
+
+	// This should msgpack-encode as a uint8 (0xcc), and not an
+	// int8 (0xd0).
+	brmd.Revision = 128
+
+	codec := kbfscodec.NewMsgpack()
+
+	// The TLF{Writer,Reader}KeyBundleIDs should be encoded as
+	// nils (0xc0) and not omitted.
```

Comment:
Can you put a comment here about how you generated these bufs, in case something changes in the future or we need to do it for a new MD version?

Reply:
done!